### PR TITLE
Update Metricbeat README.md file

### PR DESCRIPTION
**Pull request by ruflin**
* Adjusts Metricbeat README.md to be in line with other beats
* Fixes old issue link in Filebeat and Packetbeat

### CODE REVIEW

**Comment by ruflin** [comment]
@dedemorton I was thinking that in the long term we could reduce the README.md of each beat even more and point to a global README.md under to top level directory. Winlogbeat already has a shorter version.

File: metricbeat/README.md [hunk 1]
```diff
@@ -1,23 +1,31 @@
...
-* Template
+We love contributions from our community! Please read the
+[CONTRIBUTING.md](../CONTRIBUTING.md) file.
 
-Best is to start your own module as its own beat first (see below use it as a library) so you can test it and then start a discussion with our team if it would fit into Metricbeat.
+## Nightlies
 
-## Use it as library
-Metricbeat can also be used as a library so you can implement your own module on top of metricbeat and building your own beat based on it, withouth getting your module into the main repository. This allows to make use of the schedule and interfaces of Metricbeat. A developer guide and how to do this will follow soon.
+For testing purpose every day nightlies are generated which can be found [here](https://beats-nightlies.s3.amazonaws.com/index.html?prefix=metricbeat/). Please be aware that these are built on top of master and are not stable.
```
Comment (dedemorton) [inline metricbeat/README.md]: Suggest changing this sentence to say, "For testing purposes, we generate nightly builds that you can find [here]..."

Comment (dedemorton) [inline metricbeat/README.md]: One small comment, but otherwise LGTM.

Comment (ruflin) [inline metricbeat/README.md]: I suggest we change this to `For testing purposes, we generate snapshot builds that you can find ...`. The reason is that our builds are actually snapshots and not really nightly builds. Will also adjust this for the other beats in the readme.

**Comment by ruflin** [comment]
@dedemorton Please check the updated version.

**Comment by tsg** [comment]
LGTM.

File: metricbeat/README.md [hunk 2]
```diff
@@ -1,23 +1,31 @@
 # Metricbeat
 
-Metricbeat takes metrics and statistics from your systems and ships them to elasticsearch or logstash.
+Metricbeat fetches a set of metrics on a predefined interval from the operating system and services such as Apache web server, Redis, and more and ships them to elasticsearch or logstash.
```
Comment (tsg) [inline metricbeat/README.md]: Capitalize Elasticsearch and Logstash.

Comment (ruflin) [inline metricbeat/README.md]: done

**Pull request merged**